scripts/util/estimate-limits.js

```diff
@@ -105,9 +105,9 @@ setupEnvironment["maxAllowedSellers"] = async function (sellerCount = 10) {
 
   //Create DisputeResolverFee array
   const disputeResolverFees = [
-    new DisputeResolverFee(other1.address, "MockToken1", "100"),
-    new DisputeResolverFee(other2.address, "MockToken2", "200"),
-    new DisputeResolverFee(other3.address, "MockToken3", "300"),
+    new DisputeResolverFee(other1.address, "MockToken1", "0"),
+    new DisputeResolverFee(other2.address, "MockToken2", "0"),
+    new DisputeResolverFee(other3.address, "MockToken3", "0"),
   ];
```

Comment:
I get an `Error: VM Exception while processing transaction: reverted with reason string 'Non-zero dispute resolver fees not yet supported'`

when I run `npm runestimate-limits` for `maxFeesPerDisputeResolver`
I think the problem is on line 149, but I can't comment on that line in this view.

Reply:
Fixed, I think I canceled the script instead of waiting for finished the execution when I tested